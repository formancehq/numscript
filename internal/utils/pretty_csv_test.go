package utils_test

import (
	"testing"

	"github.com/formancehq/numscript/internal/utils"
	"github.com/gkampitakis/go-snaps/snaps"
)

func TestPrettyCsv(t *testing.T) {
	out := utils.CsvPretty([]string{
		"Account", "Asset", "Balance",
	}, [][]string{
		{"alice", "EUR/2", "1"},
		{"alice", "USD/1234", "999999"},
		{"bob", "BTC", "3"},
	})

	snaps.MatchSnapshot(t, out)
}
