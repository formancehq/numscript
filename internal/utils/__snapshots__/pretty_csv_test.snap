
[TestPrettyCsv - 1]
| [36mAccount[0m | [36mAsset[0m | [36mBalance[0m |
| alice   | EUR/2    | 1       |
| alice   | USD/1234 | 999999  |
| bob     | BTC      | 3       |
---
